### ISSUE TITLE
Extension server Server returned 503

### ISSUE THREAD

**Issue by souths** [Microsoft/vscode#57880]
- VSCode Version:
- OS Version:

Steps to Reproduce:

1.
2.

Does this issue occur when all extensions are disabled?: Yes/No

**Comment by rodoaravena** [comment]
I have the issue, I can't access to the marketplace and several services of Visual Team Services

**Comment by souths** [comment]
1.Search for anything
2.Server returned 503

**Comment by xuntee** [comment]
我以为被中国防火墙屏蔽了   原来都这样 不能访问
(translate)
I thought it was blocked by the Chinese firewall. It turned out to be inaccessible.

**Comment by kaushiksanthanam** [comment]
Not able to access the marketplace.

**Comment by TheColorRed** [comment]
When the marketplace is inaccessible, there should be a message displaying so in the editor, otherwise it just feels broken. Instead of displaying **No extensions found**, maybe it should display: 

**The marketplace is inaccessible at the moment, please try again later**

or something similar...

**Comment by igorbb** [comment]
https://blogs.msdn.microsoft.com/vsoservice/?p=17405

Also this seems to be a duplicate of :

#57851
